test/JDBC/expected/fts-contains-vu-verify.out

```diff
@@ -2636,6 +2636,422 @@ int#!#text
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near 'where' at line 1 and character position 41)~~
+
+
+select t.name from test_special_char_t t where contains(.name, 'one');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error at or near ".")~~
+
+
+select t.name from test_special_char_t t where contains(t., 'one');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near 'where' at line 1 and character position 41)~~
+
+
+select t.name from test_special_char_t t where contains(t.., 'one');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near 'where' at line 1 and character position 41)~~
+
+
+select t.name from test_special_char_t t where contains(, 'one');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near 'where' at line 1 and character position 41)~~
+
+
+select t.name from test_special_char_t t where contains(t.txt, 'one');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: column t.txt does not exist)~~
+
+
+select t.name from test_special_char_t t where contains(b, 'one');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: column "b" does not exist)~~
+
+
+select t.name from test_special_char_t t where .t.name = 'one';
+go
+~~START~~
+text
+one
+~~END~~
+
+
+select t.name from test_special_char_t t where contains('t.name', '"one two"');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: syntax error near 'where' at line 1 and character position 41)~~
+
+
+select t.name from test_special_char_t t where contains("t.name", '"one two"');
+go
+~~START~~
+text
+one two three
+one~two
+one!two
+one@two
+one#two
+one$two
+one%two
+one^two
+one&two
+one*two
+one-two
+one+two
+one=two
+one\two
+one|two
+one;two
+one:two
+one<two
+one>two
+one.two
+one?two
+one/two
+one :) two
+one @ @ @ @ two
+one   @ two    ^ three
+one     @    two
+one @ two @ three @ four
+one # two ' three ` four
+one    @ two    $ three ^ four * five
+one $ two % three * four * five
+one # two : three < four > five
+: one two three
+one: two
+:one two:
+~~END~~
+
+
+select name as txt from test_special_char_t where contains(txt, 'hello');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: column "txt" does not exist)~~
+
+
+use fts_test_db;
+go
+
+select * from fts_test_db.new_schema_fts.test t where contains(t.name, 'one');
+go
+~~START~~
+int#!#text
+1#!#one two three
+~~END~~
+
+
+select * from fts_test_db.new_schema_fts.test t where contains(fts_test_db.new_schema_fts.t.name, 'one');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: missing FROM-clause entry for table "new_schema_fts")~~
+
+
+select * from fts_test_db.new_schema_fts.test where contains(name, 'one');
+go
+~~START~~
+int#!#text
+1#!#one two three
+~~END~~
+
+
+select * from fts_test_db.new_schema_fts.test where contains(fts_test_db.new_schema_fts.test.name, 'one');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: missing FROM-clause entry for table "new_schema_fts")~~
+
```

Comment:
we do not have test for schema.table.column in contains clause

Reply:
done